Samples/Cordova/ExampleProject/platforms/android/app/src/main/java/com/clevertap/cordova/CleverTapPlugin.java

```diff
@@ -169,27 +166,18 @@ public void run() {
                     }
                 }
 
-                final String json = "{'notification':" + data.toString() + "}";
+                final String json = "{'notification':" + data + "}";
                 cordova.getActivity().runOnUiThread(new Runnable() {
```

Comment:
I think there is no real difference between `cordova.getActivity().runOnUiThread` and the `getView().post` inside the emitter. You can remove the this runnable and just leave the emitter.sendEvent

Reply:
FIxed in [d77b2c6](https://github.com/CleverTap/clevertap-cordova/pull/259/commits/d77b2c683560356e6a2f25028c2ae1f77ce70557)